CONTRIBUTING.md

```diff
@@ -88,17 +80,50 @@ yarn rwt link ../path/to/redwood
 
 Your Redwood App isn't using the packages in `node_modules/@redwoodjs` anymore. It's now using the packages in the new workspace with your local changes. You can even install packages or upgrade dependencies &mdash; it's really that simple.
 
-`rwt link` won't return you to the command line. Once it stops outputting to the terminal, you can open a new tab in your terminal, cd to `redwood-app` and launch your Redwood App from there. As you make changes in `redwood`, you will see those changes immediately in the behavior of your Redwood App.
+`rwt link` won't return you to the command line. Once it sets everything up you should see:
+```terminal
+   ┌─────────────────────────────────────────────────────────────┐
+   │                                                             │
+   │   🚀 Go Forth and Contribute!                               │
+   │                                                             │
+   │   🔗  Your project is linked!                               │
+   │                                                             │
+   │   Contributing doc: https://redwoodjs.com/docs/contributing │
+   │                                                             │
+   └─────────────────────────────────────────────────────────────┘
+```
 
-When you're done, go back to your `rwt link` tab and ctrl-c to quit. `rwt link` will issue a `yarn rwt unlink` comand. This will restore your Redwood App to its original state, reverting it to the version of Redwood that it originally had installed.
+you can open a new tab in your terminal, cd to `redwood-app` and launch your Redwood App from there. As you make changes in `redwood`, you will see those changes immediately in the behavior of your Redwood App.
+
+When you're done, go back to your `rwt link` tab and ctrl-c to quit. `rwt link` will ask you to run `yarn rwt unlink` comand. This will restore your Redwood App to its original state, reverting it to the version of Redwood that it originally had installed.
 
 Next time you want to contribute, just run `yarn rwt link` again!
 
+
+*Flags for rwt link command*
+| Flag | Description                              |
+| :------------------ | :--------------------------------------- |
+| `RW_PATH`              | Path to the framework repo. Required if RW_PATH environment var not set                         |
+| `--clean, -c`       | Clean the framework repo before linking? Set this to false to speed things up if you're sure your build folders are clean [default: true]                 |
+| `--watch, -w`           | Set this to false, if you just want to link once, and not watch for file changes [default: true]      |
```

Comment:
- don't think we need commas here
```suggestion
| `--watch, -w`           | Set this to false if you just want to link once and not watch for file changes [default: true]      |
```

Reply:
lol British English vs American English? I'll accept